src/makernote_int.cpp

```diff
@@ -1190,6 +1190,18 @@ namespace Exiv2 {
         };
         return std::find(std::begin(models), std::end(models), getExifModel(pRoot)) != std::end(models) ? 0 : -1;
     }
+    int sony2FpSelector(uint16_t /*tag*/, const byte* /*pData*/, uint32_t /*size*/, TiffComponent* const pRoot)
+    {
+        // Not valid for models beginning
+        const char* models[] = { "SLT-", "HV", "ILCA-" };
+
+        std::string model = getExifModel(pRoot);
+        for (auto&& m : models) {
+            if (model.find(m) == 0)
```

Comment:
It's frustrating that [`starts_with`](https://en.cppreference.com/w/cpp/string/basic_string/starts_with) is a C++20 feature, so I don't think we can use it yet. I would suggest adding a copy of [`startsWith`](https://github.com/Exiv2/exiv2/blob/6798be637f9b36675edcae4504e1dee18091462f/src/epsimage.cpp#L138) to this file, with a TODO comment to replace it with `starts_with` in the future.

Reply:
I have added the function.
Thanks.